packages/wrangler/src/dev.tsx

```diff
@@ -497,9 +497,10 @@ export async function startApiDev(args: StartDevOptions) {
 			configParam
 		);
 
+		const disableBundling = configParam.no_bundle ?? !args.bundle;
```

Comment:
args.bundle should override configParam.no_bundle, so this order might be wrong

Reply:
Gah